Bottom: 50f94433013357e55892fe826c7e6fb0a2ba67e3
Top:    b84126088f51fe4035f41117bdf1f2add9af0794
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-05 18:44:15 +0100

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 3747233..1c755f7 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
